Add config flow for Sun (#22061)
home-assistant/home-assistant.github.io: source/_integrations/sun.markdown

  - '@Swamp-Ig'
ha_iot_class: Calculated
ha_domain: sun
ha_config_flow: true
---

The sun integration will use the location as
{% my general title="configured in your Home Assistant configuration" %} to
track if the sun is above or below the horizon. The sun can be used within

> automation as

Reviewer: ```suggestion
automations as
```

> [a trigger with an optional offset to simulate dawn/dusk][sun_trigger]or as

Reviewer: ```suggestion
[a trigger with an optional offset to simulate dawn/dusk][sun_trigger] or as
```

  - '@Swamp-Ig'
ha_iot_class: Calculated
ha_domain: sun
ha_config_flow: true
---

> The sun integration will use your current location to track if the sun is above or

Reviewer: We should rephrase, the integration will not use "your current location", it will use the location of the Home Assistant server/installation as configured by the user.

Author: Adjusted.

[sun_trigger]: /docs/automation/trigger/#sun-trigger
[sun_condition]: /docs/scripts/conditions/#sun-condition

## Configured by default

This integration is by default configured and installed, and you don't need
to configure it yourself, unless you've disabled or removed the
[`default_config:`](/integrations/default_config/) line from your
YAML configuration.

If that is the case, you can configure it as described in the next paragraphs.

{% include integrations/config_flow.md %}

## YAML Configuration

Alternatlively, this integration can be configured and set up manually via YAML

> as well. To enable the sun integration in your installation, add the

Reviewer: ```suggestion
instead. To enable the sun integration in your installation, add the
```

Alternatlively, this integration can be configured and set up manually via YAML
as well. To enable the sun integration in your installation, add the
following to your `configuration.yaml` file:

```yaml
# Example configuration.yaml entry
sun:
```

{% configuration %}
elevation:
  description: "The (physical) elevation of your location, in meters above sea level. Defaults to the `elevation` in `configuration.yaml`, which is retrieved from Google Maps if not set."
  required: false

>   type: integer

Reviewer: Oops, this was still documented 🤦

Should we add a note about how to actually configure location and elevation though?

Author: This is part of the onboarding processes and UI core configuration for a long time already. You actually can't skip it.

Author: Added my link to introduction text
